CryptoTechReminderSystem.Test/ShameLateDevelopersTests.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Linq;
+using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.Test.TestDouble;
+using CryptoTechReminderSystem.UseCase;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace CryptoTechReminderSystem.Test
+{
+    public class ShameLateDevelopersTests
+    {
+        [Test]
+        public void CanGetLateDevelopers()
+        {
+            var remindDeveloperSpy = new RemindDeveloperSpy();
```

Comment:
Lots of duplication in these tests

Reply:
We should never have _any_ duplication anywhere. Duplication is never ok!

Reply:
Fixed in https://github.com/chubberlisk/crypto_tech/pull/17/commits/27d3216d3861aeaa9ced93e427a2166569d99f94